### PR TITLE
Alert button

### PR DESCRIPTION
**Pull request by dipen2000**
Created the alert and button component please review my code and give feedback

### CODE REVIEW

File: CSS/documentation.css [hunk 1]
```diff
@@ -119,14 +144,37 @@ main{
     font-size: 1.6rem;
 }
 
+/************************** avatar component documentation styles ********************/
 
+.component-display{
+    /* border : 3px solid red; */
```
Comment (rahulyadav139) [inline CSS/documentation.css:150]: remove unnecessary comments, use rem instead of px.

**Review by rahulyadav139 — APPROVED** [review]

File: components/badges/badge.css [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+.left{
+    left : 0;
+}
+
+.bottom{
+    bottom : 0;
+}
+
+.top{
+    top : -1rem;
```
Comment (dev-ashishsethi) [inline components/badges/badge.css:22]: instead of using negative value in top, use positive value in bottom

**Review by dev-ashishsethi — APPROVED** [review]

File: components/alert/alert.css [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+.alert-warning-solid{
+    color : var(--warning-alert-color);
+    background-color : var(--warning-alert-background);
+    border : 3px solid var(--warning-alert-background);
+}
+
+.alert-warning-outline{
+    color : var(--warning-alert-color);
+    border : 3px solid var(--warning-alert-color);
```
Comment (SinchanaMS) [inline components/alert/alert.css:52]: Increase the contrast on the alerts for better readability.

image

**Review by j24m — APPROVED** [review]
Actually everything looks good to go for now, but the minor changes suggested by people are good practices to follow so you can do the same.  Good work !! :)

File: components/alert/alert.css (hunk 3 above)
Comment (m26unkwn) [reply components/alert/alert.css:52]: +1

**Review by m26unkwn — APPROVED** [review]
Everything is good except the alert colors.
Just Change that then good to merge.

**Pull request merged**